src/lib/modules/Items/backoffice/ItemListEntry.js

```diff
@@ -167,7 +204,11 @@ export class ItemListEntry extends Component {
             </Grid.Column>
           </Grid>
         </Item.Content>
-        <div className="pid-field">#{item.metadata.pid}</div>
+        {checkIn ? (
+          <div className="pid-field pid-field-top">#{item.metadata.pid}</div>
```

Comment:
actually it would be enough if you just removed all the classes here, without creating a new one